src/main/java/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package strategy.impl;
+
+import strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line to follow the checklist guidelines.